### DISCUSSION TITLE
cesium光照到模型上产生异样的波纹（粼片），咋回事

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#594] [Q&A]
![3ff04191646d9aca2dba06b2461afe2](https://github.com/user-attachments/assets/7c1b9f00-9cd8-4fb4-8a62-659779ec7563)
如图，在夜间的情况更加明显，这个模型上的白色波纹到底是什么，如何消除？？

**Comment by zouyaoji** [comment]
搞不懂了，建议发个重现的链接，看有没有人明白咋回事
